src/main/java/net/masterthought/cucumber/CucumberReportGeneratorMojo.java

```diff
@@ -86,15 +95,18 @@ public void execute() throws MojoExecutionException {
             Configuration configuration = new Configuration(outputDirectory, projectName);
             configuration.setBuildNumber(buildNumber);
             configuration.setParallelTesting(parallelTesting);
+            if (classifications != null && !classifications.isEmpty()) {
```

Comment:
why not at once https://commons.apache.org/proper/commons-collections/javadocs/api-3.2.2/org/apache/commons/collections/CollectionUtils.html#isEmpty(java.util.Collection) ?